packages/component-footer/src/core/constants/index.js

```diff
@@ -0,0 +1,3 @@
+import { FOOTER_URLS, LEGAL_URLS } from "./constants";
+
+export { FOOTER_URLS, LEGAL_URLS };
```

Comment:
export * from "./constants";

Reply:
Actually is there a better file name/ path? this seems redundant
- `/core/constants`
  - `/core/constants/constants.js`
  - `/core/constants/index.js`